weave-js/package.json

```diff
@@ -135,6 +135,7 @@
     "react-vega": "^7.4.1",
     "react-virtualized": "^9.10.1",
     "react-virtualized-auto-sizer": "^1.0.6",
+    "react-vis": "^1.12.1",
```

Comment:
react-vis hasn't been meaningfully updated in a few years. Recharts and Nivo are decent options. Maybe one of them is fairly compatible with minimal effort to change. Totally optional though